Core/clim-basic/recording.lisp

```diff
@@ -1937,57 +1934,57 @@ add output recording facilities. It is not instantiable."))
               (stream-current-text-output-record stream) record)))
     record))
 
-(defmethod stream-close-text-output-record
-    ((stream standard-output-recording-stream))
-  (let ((record (stream-current-text-output-record stream)))
-    (when record
-      (setf (stream-current-text-output-record stream) nil)
-      #|record stream-current-cursor-position to (end-x record) - already done|#
-      (stream-add-output-record stream record))))
-
-(defmethod stream-add-character-output
-    ((stream standard-output-recording-stream)
-     character text-style width height baseline)
-  (add-character-output-to-text-record
-   (stream-text-output-record stream text-style)
-   character text-style width height baseline))
+(defmethod stream-close-text-output-record ((stream standard-output-recording-stream))
+  (when-let ((record (stream-current-text-output-record stream)))
+    (setf (stream-current-text-output-record stream) nil)
+    #|record stream-current-cursor-position to (end-x record) - already done|#
```

Comment:
@dkochmanski is it a leftover note to self?

Reply:
no, this comment was already in this method. since I don't understand it I didn't remove it.